packages/vue-components/src/panels/PanelBase.js

```diff
@@ -145,10 +145,8 @@ export default {
         requestAnimationFrame(() => {
           // To enable behaviour of auto window scrolling during panel collapse
           if (this.$el.getBoundingClientRect().top < 0) {
-            const headerHeight = jQuery('header[sticky]').height() || 0;
-            jQuery('html').animate({
-              scrollTop: window.scrollY + this.$el.getBoundingClientRect().top - headerHeight - 3,
-            }, 500, 'swing');
+            const headerHeight = document.querySelector('header[sticky]').length || 0;
```

Comment:
@yucheng11122017 so what's the answer to the above query?

Reply:
I changed it to 

```
  const headerComputedStyle = window.getComputedStyle(headerEl, null);
  headerHeight = headerEl.clientHeight - parseFloat(headerComputedStyle.paddingTop)
      - parseFloat(headerComputedStyle.paddingBottom) || 0;
```
because jquery height() calculates the height without the padding but clientHeight includes the padding.